docs/Tutorials/MathComponent/Tutorial.md

```diff
@@ -101,15 +101,15 @@ in the [F Prime git repository](https://github.com/nasa/fprime).
 You may also wish to work through the Getting Started tutorial at
 `docs/GettingStarted/Tutorial.md`.
 
-**Git branch:** This tutorial is designed to work on the branch `release/v3.0.0`.
+**F' Version:** This tutorial is designed to work with release `v3.0.0`.
 
 Working on this tutorial will modify some files under version control in the
 F Prime git repository.
 Therefore it is a good idea to do this work on a new branch.
 For example:
 
 ```bash
-git checkout release/v3.0.0
+git checkout -b release/v3.0.0 v3.0.0
```

Comment:
I think you want this to be a one-liner:

```
git checkout -b math-tutorial v3.0.0
```

or a two step:

```
git checkout v3.0.0
git checkout -b "math-tutorial
```

No need to create a branch `release/v3.0.0` only to throw it away in the next step.